BUILD.md

```diff
@@ -143,6 +144,9 @@ GCC 4.x and below are not supported.
 |**Node.js**|--build_nodejs|Build Node.js binding. Implies `--build_shared_lib`|
 
 ---
+## Reduced Operator Kernel Build
+Custom Reduced Operator Kernel builds can provide smaller binary sizes - [see instructions](./docs/Reduced_Operator_Kernel_build.md). Note: if you are looking for mobile-compatible builds, refer to the [Android](#Android) and [iOS](#iOS) sections on this page.
+
```

Comment:
The [ONNX Runtime Mobile doc](docs/ONNX_Runtime_for_Mobile_Platforms.md) for the reduced size packages overlaps with this. Its not limited to Android and iOS.

Reply:
The Reduced Operator Kernel build can be useful in other scenarios than where the ONNX Runtime Mobile build would be used, which is why I called out both. Is the ONNX Runtime Mobile doc linked from anywhere to help with discoverability?